source/framework/vx_node.c

```diff
@@ -252,13 +277,25 @@ vx_status ownNodeKernelValidate(vx_node node, vx_meta_format meta[])
             for (i = 0; i < num_params; i ++)
             {
                 meta[i]->type = node->kernel->signature.types[i];
+                if ((NULL != node->parameters[i]) &&
+                    ((vx_bool)vx_true_e == node->parameters[i]->is_virtual))
+                {
+                    node->parameters[i]->is_accessible = (vx_bool)vx_true_e;
```

Comment:
@jessegv  to be honest I do not remember if we discover it via a test case or during a review,

Reply:
When reviewing this issue further on our end, it appears that this is a valid bug fix, so we have filed bug TIOVX-2071 and will add it to the comment message when we are done with our rebase.

Regarding the initialize callback, this will also be needed but we will fix it separately on our end once we push it along with a dedicated test case.